src/Components/WebAssembly/WebAssembly/src/Services/WebAssemblyNavigationManager.cs

```diff
@@ -24,12 +24,8 @@ internal WebAssemblyNavigationManager()
         {
```

Comment:
Is this parameterless constructor still used (e.g., in tests)?

 * If not, or if there aren't many tests that use it, let's consider removing the parameterless constructor
 * If it is, could we make the access modifiers the same for both constructors? It doesn't seem like one of them should be `internal` and the other `public`. Since the type itself is `internal`, the constructors can be `public`. I know this isn't something you did in this PR but just suggesting it as a minor improvement while we're looking at it :)

Reply:
Noted.

I've gone ahead and removed it since there are no references to this.